src/pygeosx/unitTests/twoLayers2000x2000x2000.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Solvers>
+    <!-- define the solver -->
+    <!-- we'll have to discuss the cflFactor with Randy -->
+    <!-- define the source coordinates -->
+    <!-- define the time source frequency -->
+    <!-- define the receiver coordinates -->
+    <AcousticSEM
+      name="acousticSolver"
+      cflFactor="0.8"
+      discretization="FE1"
+      targetRegions="{Region,Region1}"
+      sourceCoordinates="{{1000.1,1000.1,30.1}}"
+      timeSourceFrequency="10.0"
+      receiverCoordinates="{{550.1,1000.1,30.1}}"
+      outputSeismoTrace="0"
+      dtSeismoTrace="0.001" />  
+
+  </Solvers>
+
+  <!-- hexahedral mesh generated with vtk from regular IJK grid -->
+  <Mesh>
+    <InternalMesh
+      name="mesh"
+      elementTypes="{ C3D8 }"
+      xCoords="{ 0, 2000 }"
+      yCoords="{ 0, 2000 }"
+      zCoords="{ 0, 500,2000 }"
+      nx="{ 100 }"
+      ny="{ 100 }"
+      nz="{ 100,300}"
```

Comment:
Are you really running a unit test with ~1e6 elements?
How much time does it last for CPU? And for GPU?

Reply:
As written below, I have removed this test and simply updated the current wave propagation unit test.
We will need to discuss how we can effectively write PyGEOSX integrated tests though, I'll maybe create a dedicated PR. Thanks for the feedback!